xml/System.Globalization/CompareInfo.xml

```diff
@@ -3097,14 +3107,24 @@ Use <xref:System.Globalization.CompareInfo.GetSortKeyLength(System.ReadOnlySpan{
         <Parameter Name="matchLength" Type="System.Int32" RefType="out" Index="3" FrameworkAlternate="net-5.0" />
       </Parameters>
       <Docs>
-        <param name="source">To be added.</param>
-        <param name="suffix">To be added.</param>
```

Comment:
Here's the weird line where the latest CI job showed a `suffix` parameter.

Reply:
@joelmartinez We're a bit confused why mdoc didn't realize the `suffix` parameter was renamed to `prefix`, and instead just added `prefix` as an additional parameter. The deletion of `suffix` here is something I did manually.

Reply:
@gewarren mdoc will remake the `Parameter` elements because those come directly from the metadata so it remakes them every time ... but in general it's a bit more tentative with removing elements in the `Docs` node, because those can contain user-written content, so in cases like this it's best just to adjust manually as you've done here

Reply:
Would that even have shown up in the rendered page if I hadn't removed it?

Reply:
@gewarren no I believe it wouldn't, because the rendering is driven by the Parameter entries ... it also shouldn't affect the intellisense, because it would just kind of be ignored, so even if we leave it it wouldn't be world-ending :)